helix-view/Cargo.toml

```diff
@@ -38,6 +38,8 @@ log = "~0.4"
 
 which = "4.2"
 
+notify = "=5.0.0-pre.11"    # check that it builds on NetBSD before upgrading
```

Comment:
Is there a reason we need to use a pre-release version instead of current stable? (Which I believe is 4.x)

Reply:
Last time I looked at it, it seemed like they were supporting both versions and 5 seemed stable enough maybe